jest.config.js

```diff
@@ -4,7 +4,7 @@
 module.exports = {
   coverageDirectory: 'coverage',
   collectCoverageFrom: ['<rootDir>/src/**/*.{js,jsx}'],
-  setupTestFrameworkScriptFile: '<rootDir>/test/setup.js',
+  setupFilesAfterEnv: ['<rootDir>/test/setup.js'],
```

Comment:
Is it a new jest property? 🤔

Reply:
Yep. The previous one, which accepted only a single path, was deprecated in favor of this new options that allows you to pass an array of paths.

For reference, see the note on:
https://jestjs.io/docs/en/configuration.html#setupfilesafterenv-array